src/components/ui/Skeleton/Skeleton.tsx

```
@@ -4,14 +4,24 @@ import { customClassSwitcher } from '~/core';
 import { clsx } from 'clsx';
 const COMPONENT_NAME = 'Skeleton';
 
-const Skeleton = ({ loading = true, className = '', customRootClass = '', children, ...props }:any) => {
+export type SkeletonProps = {
+    loading:boolean;
+    className?:string;
+    customRootClass?:string;
+    children:React.ReactNode;
+    height:Number;
```

Biome (1.9.4)

[error] Don't use 'Number' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'number' instead


(lint/complexity/noBannedTypes)